### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by neonm8** [D3uxx/csgojackpotV2#2]
i was just wondering when the bot will be out.

**Comment by c0deina** [comment]
I will release it this week.

**Comment by neonm8** [comment]
ok sounds good does it work with 500$ pots

**Comment by c0deina** [comment]
Yes, I saw sites with this same source with more than $650 pots.

**Comment by neonm8** [comment]
D3uxx i will pay u if u can help me setup a site when your not busy do have Jabber/XMPP?

**Comment by N3mE** [comment]
here is my site> http://csgocast.com and i have the bot script but i get a fuking idiot error in console when sameone trade with the bot>  http://i.imgur.com/XF2ecUG.png

Is not my console, but its same error!

**Comment by Oliten** [comment]
Oh shit :'( I am looking for this bot ! Courage man and thx for your work !

**Comment by DinashRisaN** [comment]
When does it come out?

The bot :) @D3uxx

**Comment by ghost** [comment]
@D3uxx The source you released is just awesome ! However, without a working bot API, it's really hard to setup a jackpot website. Hope you still plan to release one ! Thank you :)

**Comment by jdtoole** [comment]
hi the bot is here with script for free, i have it:
https://www.nulled.io/topic/47099-original-fire-csgo-japot-script-how-to-sql/?hl=%2Bcsgo+%2Bjackpot

https://www.nulled.io/topic/90418-updated-cs-go-jackpot-steam-bot/#entry2090548
